lib/src/onePlace.dart

```diff
@@ -25,6 +28,11 @@ class OnePlace extends StatefulWidget {
 }
 
 class _OnePlaceState extends State<OnePlace> {
+  @override
+  void initState() {
+    super.initState();
+  }
+
```

Comment:
Was this for something that is gone now?

Reply:
Actually, yeah I think I was planning on adding another thing but thought it would just be unnecessary.